### ISSUE TITLE
db.collections and db.listCollections returning the same?

### ISSUE THREAD

**Issue by ciscoheat** [arangodb/arangojs#244]
It seems like they both return what I'm expecting `listCollections` to return. I can't use any of the collection methods on the returned objects when using `collections`.

**Comment by pluma** [comment]
I can't reproduce this problem. `collections` returns a list of collection instances for me.

Could you show the code you are using?

**Comment by ciscoheat** [comment]
It's very simple:

```js
var db = new require("arangojs").Database({ url : "http://root:@localhost:8529"});

db.collections(function(err, collections) {
	async.each(collections,function(c, done) {
		// Cannot call truncate directly here, get this error:
		// TypeError: c.truncate is not a function
		//
		// Need to call db.collection first:
		db.collection(c.name).truncate(function(err1, info) {
			if(err1 != null) {
				done(err1);
				return;
			}
			done(null);
		});
	},function(err2) {
		// ...
	});
});
```

I'm using arangojs 5.0.0 and ArangoDB 3.0.0-rc2. I should have said that yesterday, but it was very late...!

**Comment by pluma** [comment]
Interesting. The problem only seems to occur when using the callback API but not the promise API.

**Comment by pluma** [comment]
I've published 5.0.1 which fixes this. Thanks for pointing this out.

**Comment by ciscoheat** [comment]
No problem, thanks for the quick fix! :) Confirmed that it's working for me.
